docs/docs/user-guide/configuring-cli.md

```diff
@@ -6,16 +6,26 @@ To reset settings to their default values, remove them from the configuration fi
 
 ## Configuring settings
 
-You can configure the specific setting using the `cli config set` command. For example, to configure CLI to automatically show help when executing a command failed, execute:
+You can configure the specific setting using the `cli config set` command. 
```

Comment:
We don't need to show a sample of all different configuration settings. This paragraph was meant to illustrate how the command works. The available options and their purpose are listed in the table below. That way we can keep them in one place.